config.js.example

```diff
@@ -22,5 +22,5 @@ global.ALLOWED_CLIENTS = [
   // W3C:
   /https?:\/\/(www\.)?w3c?\.org/i,
   // GitHub:
-  /https?:\/\/github\.com/i
+  /https?:\/\/.*\.github\.io/i
```

Comment:
I must admit I haven't digged into the CORS part of Echidna's code so far. Giving it a look now, I'm  confused (and tired :-) ) if we are safe or not. I'd be more comfortable if someone else than me can give a review before merging (plus there would be a conflict of interest :p), but decision to merge comes down to you so it's your call!

I guess a better solution would be to write some tests around the CORS snippet of code, if you are feeling bored... :p